.azure-pipelines/prValidate.yml

```diff
@@ -29,7 +29,7 @@ pool:
   vmImage: windows-latest
 
 steps:
-- template: templates/checkout-and-credscan.yml
-- template: templates/install-java.yml
-- template: templates/build-and-coverage.yml
+- template: templates/build/checkout-and-credscan.yml
```

Comment:
I'm curious to why we still need 2 pipelines? Couldn't we just use one and have conditions on the trigger type for the templates we want to skip?

Reply:
I don't believe we plan on having it moving forward, this is essentially doing everything that the build pipeline does anyways. I made these changes because in order to auto merge here we needed the pr validate pipeline to succeed.

Reply:
I have disabled the old protection rule that requires this pipeline, I'll enable a new one on the new pipeline once it gets all the way to master.